server/templates/diff.html

```diff
@@ -19,7 +19,7 @@
       </div>
       {% endif %}
     </div>
-    <div class="comment-body markdown-body">{{ comment.message | markdown }}</div>
+    <div class="comment-body markdown-body">{{ comment.message | safe | markdown }}</div>
```

Comment:
This seems to cause the problem but at the cost of breaking valid characters like `'`.